src/gen_rules.ml

```diff
@@ -805,6 +844,10 @@ module Gen(P : Params) = struct
      | Aliases                                                         |
      +-----------------------------------------------------------------+ *)
 
+  let interpret_locks ~dir ~scope locks =
+    List.map locks ~f:(fun s ->
+      Path.relative dir (SC.expand_vars sctx ~dir ~scope s))
+
```

Comment:
This function already exists in this file, is this a typo?

Reply:
oops, removed